node/node.go

```diff
@@ -302,9 +303,15 @@ func NewNode(mode Mode, cfg *config.Node, version string) (*Node, error) {
 			cfg.Coordinator.EthClient.Keystore.Password); err != nil {
 			return nil, tracerr.Wrap(err)
 		}
+		//Swap bjj from bigEndian to LittleEndian
+		bjjRaw, _ := cfg.Coordinator.FeeAccount.BJJ.MarshalText()
+		var bjj babyjub.PublicKeyComp
+		if err := bjj.UnmarshalText(common.SwapEndianness(bjjRaw)); err != nil {
+			return nil, tracerr.Wrap(err)
+		}
```

Comment:
You can only make a copy of the bit array without unmarshal
```suggestion
		copy(bjj[:], bjjRaw[:])
```

Reply:
Yes, however is more legible to write it in this way